src/uu/chcpu/src/errors.rs

```diff
@@ -0,0 +1,106 @@
+// This file is part of the uutils hostname package.
+//
+// For the full copyright and license information, please view the LICENSE
+// file that was distributed with this source code.
+
+use std::fmt;
+use std::path::PathBuf;
+
+use uucore::error::UError;
+
+#[derive(Debug)]
```

Comment:
could you please move to thiserror ?

Reply:
I assume you're referring to the `thiserror` crate.

Reply:
correct

Reply:
Sure.